CHANGELOG.md

```diff
@@ -2,6 +2,68 @@
 
 All notable changes to this project are documented in this file.
 
+## 1.38.0
+
+**Release date:** 2024-07-30
+
+This release comes with several fixes and improvements. There is a new [Keptn
+metrics provider](https://docs.flagger.app/usage/metrics#keptn) that can be used
+for flexible grading logic and analysis.
```

Comment:
Did you mean flexible gating?

Reply:
i just picked that up from https://docs.flagger.app/usage/metrics#keptn. happy to change it to gating as well.

Reply:
All good!